### DISCUSSION TITLE
/sethome overwrites currently set homes.

### DISCUSSION THREAD

**Discussion by accLarson** [EssentialsX/Essentials#5229] [Ideas and Feature Suggestions] [answered]
I've had incidents on my server throughout the years of players either accidentally or unknowingly overwriting their currently set home(s). 

Should this be prevented?

`/sethome` and `/sethome <name>` could check if the home being set is already set and if so, display an error/suggestions "use /forcesethome to overwrite previously set homes"

Maybe this could be implemented with a config setting defaulted to false.
`require-forcesethome-to-overwrite-homes: false`

EssentialsX's home system is the standard and most well known home system on Minecraft  servers so most people understand that `/sethome` will overwrite, but for those who make a mistake or don't understand, the effect is potentially game-ruining.

**Comment by SrBedrock** [comment, marked as answer]
There is a setting to activate this function, the player will be asked to resend the command to confirm the `/sethome`.

path: plugins/Essentials/config.yml
```yml
# Should players be asked to provide confirmation for homes which they attempt to overwrite?
confirm-home-overwrite: true
```

**Reply by accLarson** [reply]
👀 thanks, I guess I should have done more research prior to posting.
